rootfs/etc/nginx/lua/test/configuration_test.lua

```diff
@@ -0,0 +1,131 @@
+local configuration = require("configuration")
+local unmocked_ngx = _G.ngx
+
+function get_mocked_ngx_env()
+    local _ngx = {
+        status = 100,
+        print = function(msg) return msg end,
```

Comment:
why are you mocking this?

Reply:
I checked the API spec and the `timer` phase was not in the list of valid contexts: rewrite_by_lua*, access_by_lua*, content_by_lua* . However, I removed the mock and the tests pass so it looks like it works fine, with the only difference that executing the tests now print to stdout: 
``` ●●●●●●●●●●●Only POST and GET requests are allowed!●Only POST and GET requests are allowed!●●●●``` since we're logging:`Only POST and GET requests are allowed!` on two different tests.

Reply:
the print message is fine, we can mock the log message in one place when test starts but it's not the concern of this PR